### PR TITLE
Calculate homerowrk Lukasz K/Nanautzin

### PR DESCRIPTION
**Pull request by lukasz-kukulka**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @lukasz-kukulka

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lukasz-kukulka for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add"){
+        return std::to_string(first + second);
+    } else if (command == "subtract"){
+        return std::to_string(first - second);
+    } else if (command == "multiply"){
+        return std::to_string(first * second);
+    }else if (command == "divide"){
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:11]: If you return variable here, you don't need to use `else if`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+        if (first == 0 || second == 0){
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:12]: You can divide 0/ 10 its 0 and this is correct answer :)

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**